common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationAwareUTF8String.java

```diff
@@ -680,21 +621,21 @@ private static void appendTitleCasedCodepoint(
       boolean precededByCasedLetter,
       UTF8String target,
       int offset) {
-    if(isAfterAsciiSpace) {
-      // Titlecasing a character if it is in the beginning of a new word.
+    if (isAfterAsciiSpace) {
+      // Title-casing a character if it is in the beginning of a new word.
```

Comment:
nit: "Title-casing" or "titlecasing" are equally good for me. Just wanted to clarify this.

Reply:
Thanks for the clarification!